### PR TITLE
BIOIN-1755-move-single-cell-to-ugbio-utils

### PR DESCRIPTION
**Pull request by inbalzelig-ugi**
Remove single cell related files, reorder some utils and move them to ugbio_core

### CODE REVIEW

File: ugvc/pipelines/single_cell_qc/test/resources/insert_subsample.fastq.gz
Comment (avigailmo) [inline ugvc/pipelines/single_cell_qc/test/resources/insert_subsample.fastq.gz:1]: I didn't see this resource in the ugbio pr. Was this resource not used?

File: ugvc/pipelines/single_cell_qc/test/resources/single_cell_qc_stats.scRNA.applicationQC.h5
Comment (avigailmo) [inline ugvc/pipelines/single_cell_qc/test/resources/single_cell_qc_stats.scRNA.applicationQC.h5:1]: same.
I didn't see this resource in the ugbio pr. Was this resource not used?

File: ugvc/pipelines/single_cell_qc/test/resources/single_cell_qc_stats_no_shortlist.scRNA.applicationQC.h5
Comment (avigailmo) [inline ugvc/pipelines/single_cell_qc/test/resources/single_cell_qc_stats_no_shortlist.scRNA.applicationQC.h5:1]: same

File: ugvc/utils/plotting_utils.py
Comment (avigailmo) [inline ugvc/utils/plotting_utils.py:1]: this file was moved to ugbio_core, why we need it also here?

File: ugvc/utils/report_utils.py
Comment (avigailmo) [inline ugvc/utils/report_utils.py:1]: same. we should use ugbio_core.report_utils

File: ugvc/utils/sorter_utils.py
Comment (avigailmo) [inline ugvc/utils/sorter_utils.py:1]: same

File: ugvc/utils/trimmer_utils.py
Comment (avigailmo) [inline ugvc/utils/trimmer_utils.py:1]: same

**Review by avigailmo — CHANGES_REQUESTED** [review]
two issues:
1) many test resources were deleted and missing from ugbio_single_cell. Is this on purpose?
2) all _utils scripts shouldn't be in ugvc - you've moved them to ugbio_core

File: ugvc/pipelines/single_cell_qc/test/resources/insert_subsample.fastq.gz
Comment (inbalzelig-ugi) [reply ugvc/pipelines/single_cell_qc/test/resources/insert_subsample.fastq.gz:1]: It is in ugbio-utils, under src/single_cell/tests/resources/insert_subsample.fastq.gz

**Review by avigailmo — APPROVED** [review]

**Pull request merged**